inc/saiport.h

```diff
@@ -210,19 +303,24 @@ typedef enum _sai_port_ptp_mode_t
  */
 typedef enum _sai_port_interface_type_t
 {
-
     /** Interface type none */
     SAI_PORT_INTERFACE_TYPE_NONE,
 
     /** Interface type CR */
     SAI_PORT_INTERFACE_TYPE_CR,
 
+    /** Interface type CR2 */
+    SAI_PORT_INTERFACE_TYPE_CR2,
```

Comment:
to be backward compatible with enum numbering you can't insert enums in between existing ones, unless you manually set their init value explicitly

Reply:
To resolve merge conflict with master I merged the change but it took it as new change. These are not my change. Please ignore this commit.

Reply:
then please rebase your change locally on current master, squash merge and force push to this PR